### PR TITLE
[ATIP-android] Modify wait object show steps

### PR DESCRIPTION
**Pull request by jacky-young**
- Modify wait object show steps

### CODE REVIEW

**Comment by jacky-young** [comment]
@Honry please review it.

File: tools/atip/atip/common/steps.py [hunk 1]
```diff
@@ -150,4 +150,23 @@ def execute_command(context, command_line):
...
+    if get_test_platform(context) == "android":
+        (return_code, output) = context.android.doCMD(command_line)
+        if return_code == 0:
+            assert context.android.save2InfoTemp(output, result_key)
+        else:
+            assert False
+
+
+# check android adb command result
+@step(u'The "{expect}" should be in result "{result_key}"')
```
Comment (Honry) [inline tools/atip/atip/common/steps.py]: The value "{expected_value}" should be in result "{result_key}"

File: tools/atip/README.md [hunk 2]
```diff
@@ -430,6 +431,8 @@ def i_visit_url(context, url):
 @step(u'I open airplane mode')
 @step(u'I close airplane mode')
 @step(u'I execute command "{command_line}"')
+@step(u'I save command "{command_line}" result "{result_key}"')
+@step(u'The value "{expect}" should be in result "{result_key}"')
```
Comment (Honry) [inline tools/atip/README.md]: 's/expect/expected_value'

**Comment by Honry** [comment]
Please provide the UT result.

**Comment by jacky-young** [comment]
verified locally

**Comment by Honry** [comment]
LGTM

**Pull request merged**